src/styles/Search.css

```
@@ -0,0 +1,34 @@
+.search__input {
+  border-radius: 11px;
+  font-size: 10px;
+  padding: 5px 10px;
+  border: 1px solid black;
+  height: 100%;
+}
+
+.search__button {
+  font-size: 10px;
+  color: white;
+  padding: 5px 10px;
+  border-radius: 7px;
+  background-color: rgb(239, 75, 75);
+  border: 1px solid rgba(0, 0, 0, 0);
+}
```

There are a few common rules here which could be consolidated:
```suggestion
.search__input,
.search__button {
  font-size: 10px;
  padding: 5px 10px;
}

.search__button {
  color: white;
  background-color: rgb(239, 75, 75);
  border: none;
}

.search__input {
  border-radius: 11px;
  border: 1px solid black;
}
```
You could also remove the height declaration on the search input and just use `border: none` for the search button